build_winapp/appxmanifest.xml

```diff
@@ -1,14 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Package
-  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
-  xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities"
-  xmlns:uap3="http://schemas.microsoft.com/appx/manifest/uap/windows10/3"
-  xmlns:desktop="http://schemas.microsoft.com/appx/manifest/desktop/windows10">
-  <Identity
-    Name="EndlessOSFoundation.EndlessKey"
-    Version="1.0.0.0"
-    Publisher="CN=4B001A25-B854-44D9-BAD3-8AD7FC7A8761"
-    ProcessorArchitecture="x64" />
+<Package xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10" xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities" xmlns:uap3="http://schemas.microsoft.com/appx/manifest/uap/windows10/3" xmlns:desktop="http://schemas.microsoft.com/appx/manifest/desktop/windows10">
```

Comment:
I find it quite hard to believe that Microsoft's validator is so broken that it can't handle newlines in an XML element. Did you test if this is actually necessary?

Reply:
Well, simply follow the format of appxmanifest.xml generated from Visual Studio.